src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.MediaPlayer.cs

```diff
@@ -321,8 +330,12 @@ private void ThumbOnDragCompleted(object sender, DragCompletedEventArgs dragComp
 
 			if (_mediaPlayer != null)
 			{
-				_mediaPlayer.PlaybackSession.Position = TimeSpan.FromSeconds(m_tpMediaPositionSlider.Value);
+				_wasPlaying = _mediaPlayer.PlaybackSession.IsPlaying;
```

Comment:
we paused the video at the start of drag (ThumbOnDragStarted)
reading from _mediaPlayer.PlaybackSession.IsPlaying will cause the video to never resume (Play) on release
if you want to guarantee that we only change position during pause, you should do somehting like:
```cs
if (_mediaPlayer.PlaybackSession.IsPlaying)
{
	// on the off chance, ThumbOnDragStarted/ThumbOnDragCompleted desync'd
	// we need to ensure the player is paused, while we update the position.
	/* edit: idk whats the edge case here, add your explanation above, and delete this line */
	_skipPlayPauseStateUpdate = true;
	_isScrubbing = true;
	_wasPlaying = true;
	
	_mediaPlayer.Pause();
}

// continue with the rest of orginal code...
_mediaPlayer.PlaybackSession.Position = TimeSpan.FromSeconds(m_tpMediaPositionSlider.Value);
if (_wasPlaying)
{
	_mediaPlayer.Play();
}
```

Reply:
I changed the strategy.
The problem happened when the m_tpMediaPositionSlider.Value was updated.
So while the mouse is on the Thumb or Slider this value is not updated. And back to be updated when leave the slider.
And use your change for the is playing.
So the tapped worked and so did the drag.
Can you validate again?